Multisig/Data/Services/Safe Client Gateway Service/SCGModels.swift

```diff
@@ -538,6 +538,42 @@ extension SCGModels {
         let blockExplorerUrl: URL
         let nativeCurrency: Currency
         let theme: Theme
+
+        enum CodingKeys: String, CodingKey {
+            case chainId
+            case chainName
+            case rpcUrl
+            case blockExplorerUrl
+            case nativeCurrency
+            case theme
+        }
+
+        init(chainId: Int,
```

Comment:
Instead of implementing these here, I would implement extension on URL that would make it Decodable